environment.yml

```diff
@@ -3,6 +3,6 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python
-  - jupyter-book
-  - sphinx-tags
+- python
```

Comment:
This is not a meaningful change and actually leads to inconsistency in the file, since the list in the section above is indented with two spaces. In general two spaces for an indent is preferred for yaml.